pex/cli/commands/lock.py

```diff
@@ -32,13 +42,22 @@
 from pex.version import __version__
 
 if TYPE_CHECKING:
-    from typing import IO, List, Optional, Tuple, Union
+    from typing import IO, Dict, List, Optional, Tuple, Union
 
     import attr  # vendor:skip
 else:
     from pex.third_party import attr
 
 
+class FingerprintMismatch(Enum["FingerprintMismatch.Value"]):
+    class Value(Enum.Value):
+        pass
+
+    IGNORE = Value("ignore")
```

Comment:
Does ignoring in this case amount to adjusting the lockfile to use a new fingerprint? Or will it keep the old fingerprint? Should this have a different name that clarifies that?

Reply:
When you ignore the mismatch, the new fingerprint is used. I expanded the corresponding option help to document what each value does.